lib/pgvectors-docker-image-ecr-deployment-cdk-stack.ts

```diff
@@ -29,13 +30,28 @@ export class PgvectorsDockerImageEcrDeploymentCdkStack extends cdk.Stack {
                 POSTGRES_USER: props.envTyped.POSTGRES_USER,
                 POSTGRES_PASSWORD: props.envTyped.POSTGRES_PASSWORD,
                 POSTGRES_BASE_VERSION: props.envTyped.POSTGRES_BASE_VERSION,
-                POSTGRES_DB_NAME: props.envTyped.POSTGRES_DB_NAME,
+                POSTGRES_DB: props.envTyped.POSTGRES_DB,
             },
         });
 
-        new ecrDeploy.ECRDeployment(this, `${props.appName}-${props.environment}-DockerImageECRDeployment`, {
-            src: new ecrDeploy.DockerImageName(dockerImageAsset.imageUri),
-            dest: new ecrDeploy.DockerImageName(`${ecrRepository.repositoryUri}:${props.imageVersion}`),
+        const deployImageVersions = props.imageVersion === LATEST_IMAGE_VERSION ? props.imageVersion : [props.imageVersion, LATEST_IMAGE_VERSION];
+        for (const deployImageVersion of deployImageVersions) {
+            new ecrDeploy.ECRDeployment(this, `${props.appName}-${props.environment}-${deployImageVersion}-ECRDeployment`, {
+                src: new ecrDeploy.DockerImageName(dockerImageAsset.imageUri),
+                dest: new ecrDeploy.DockerImageName(`${ecrRepository.repositoryUri}:${deployImageVersion}`),
+            });
+        }
+
+        // print out ecrRepository arn
+        new cdk.CfnOutput(this, `${props.appName}-${props.environment}-ECRRepositoryArn`, {
+            value: ecrRepository.repositoryArn,
+            exportName: `${props.appName}-${props.environment}-ECRRepositoryArn`,
+        });
+
+        // print out ecrRepository repository name
+        new cdk.CfnOutput(this, `${props.appName}-${props.environment}-ECRRepositoryName`, {
+            value: ecrRepository.repositoryName,
+            exportName: `${props.appName}-${props.environment}-ECRRepositoryName`,
         });
```

Comment:
**Suggestion:** Instead of creating multiple `CfnOutput` instances for `ecrRepository` properties, consider combining them into a single `CfnOutput` instance with a JSON object to reduce redundancy.
> 🌸  **Medium Priority** 
 > [category: **`enhancement`**, importance: **`6`**]
> **Reason:** Combining multiple `CfnOutput` instances into a single one with a JSON object could improve maintainability and reduce redundancy. However, this might also reduce clarity for some deployments where distinct outputs are preferred.
 

```suggestion
        new cdk.CfnOutput(this, `${props.appName}-${props.environment}-ECRRepositoryInfo`, {
            value: JSON.stringify({
                arn: ecrRepository.repositoryArn,
                name: ecrRepository.repositoryName
            }),
            exportName: `${props.appName}-${props.environment}-ECRRepositoryInfo`,
        });
```